Kinvey.Core/packages.config

```diff
@@ -5,7 +5,6 @@
   <package id="Microsoft.Bcl.Build" version="1.0.21" targetFramework="portable-net45+win+wp80+MonoTouch10+MonoAndroid10+xamarinmac20+xamarinios10" />
   <package id="Microsoft.Net.Http" version="2.2.29" targetFramework="portable45-net45+win8+wpa81" />
   <package id="Microsoft.NETCore.Platforms" version="1.0.1" targetFramework="portable45-net45+win8+wp8" />
-  <package id="modernhttpclient" version="2.4.2" targetFramework="portable45-net45+win8+wpa81" />
```

Comment:
Not sure we can remove this yet, since this affects the currently-used `Kinvey.Core` project, and not the new .NETStandard-based `Kinvey` project.

Reply:
the `Kinvey` and `Kinvey.Core` shared the same code base, so no need to keep `modernhttpclient`